config/configmapprovider/retrieved.go

```diff
@@ -21,38 +21,42 @@ import (
 	"go.opentelemetry.io/collector/config"
 )
 
-// Retrieved holds the result of a call to the Retrieve method of a Provider object.
-// This interface cannot be directly implemented. Implementations must use the NewRetrieved helper.
-type Retrieved interface {
-	// Get returns the config Map. Should never be called after Close.
-	// Should never be called concurrently with itself or Close.
-	Get(ctx context.Context) (*config.Map, error)
-
-	// Close signals that the configuration for which it was used to retrieve values is
-	// no longer in use and should close and release any watchers that it may have created.
-	//
-	// Should block until all resources are closed, and guarantee that `onChange` is not
-	// going to be called after it returns except when `ctx` is cancelled.
-	//
-	// Should never be called concurrently with itself or Get.
-	Close(ctx context.Context) error
+// Get returns the config Map. Should never be called after Close.
+// Should never be called concurrently with itself or Close.
+//
+// Deprecated: Not needed, will be removed soon.
```

Comment:
Should we update the workflow now https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configmapprovider/provider.go#L27-L37 ?

Reply:
done.